core/src/main/java/org/kiwix/kiwixmobile/core/base/BaseActivity.kt

```diff
@@ -50,16 +48,6 @@ abstract class BaseActivity : AppCompatActivity() {
     unbinder = ButterKnife.bind(this)
   }
 
-  // TODO https://issuetracker.google.com/issues/141132133 remove this once appcompat has been fixed
-  override fun applyOverrideConfiguration(overrideConfiguration: Configuration?) {
-    if (Build.VERSION.SDK_INT in Build.VERSION_CODES.LOLLIPOP..Build.VERSION_CODES.N_MR1 &&
```

Comment:
Oh was this bug fixed?

Reply:
Apparently, the link says it's closed. I had to bump the version of `appcompat` as the tests were failing which I reckon was due to this issue.

Reply:
well groovy